Formula/ppl.rb

```diff
@@ -14,12 +16,12 @@ class Ppl < Formula
 
   depends_on "gmp"
 
-  # Fix compilation with Xcode 10
-  # Upstream commit, remove for next version
-  patch do
-    url "http://www.cs.unipr.it/git/gitweb.cgi?p=ppl/ppl.git;a=commitdiff_plain;h=c39f6a07b51f89e365b05ba4147aa2aa448febd7"
-    sha256 "6786f432784b74b81805b1d97e97cd1cc9f68653077681bb4f531466cbf8dc99"
-  end
+  # Fix build failure with clang 5+.
+  # https://www.cs.unipr.it/mantis/view.php?id=2128
+  # http://www.cs.unipr.it/git/gitweb.cgi?p=ppl/ppl.git;a=commit;h=c39f6a07b51f89e365b05ba4147aa2aa448febd7
+  # since 401 error on the `www.cs.unipr.it` links adopt the patch from macports
+  # patch reference, https://github.com/macports/macports-ports/commit/e5de9cc65a8e91fcbb9a3d90911569169f0ccf88
```

Comment:
Why not just use that as a patch then?

Reply:
We would have to patch the patch as our paths vary from MacPorts's 😅

Reply:
yeah, macports has its own installation, so I added the patch separately in here.